### PR TITLE
More translatable UI strings

### LINKED ISSUE #1458: Some strings untranslatable.
Hi there,

I've noticed some strings doesn't seem to be currently translatable (someones correspond to recent changes, but many others do not). 
Following is a list of the ones that I've managed to identify.

- "Toggle XSheet Toolbar" right click menu option
- "Timeline / Xsheet" button
- the word "Table", used in the Xsheet, the Stage Schematic node and the right click menu in the viewer.
- the word "XSheet", used in the FX Schematic node.
- the word "Output", used in the FX Schematic node.
- "Set  Automarkers" in right click menu in Xsheet.
- "untitled", used in Output options for the file name. 
- "untitled", used for untitled  scene name. 
- "inch", "pixels", "A/R" and "<custom>" in Camera Settings and Output Settings panels.
- "Range" and "Snap" in Brush Tool settings.
- "fld" in Edit Tool
- "H, "S", "V", "A", "R", "G", "B", "MyPaint Brush" in Style Editor.
- "[LEVEL]", "Palette" and "#" strings in bottom line of Style Editor.
- "Passive Pick" in Style Selector tool
- "Global Palettes", "Project Palettes" and "Colors" in Studio Palette Editor.
- "Alpha", "Full Alpha" and "Zero Alpha" in Palette Gizmo.
- "Apply and Next", "Easy Inputs", "New", "Character", "Part", "Suffix", "Enter new word" in Style Name Editor.
- Strings in User Preferences panel:    
"Default Import Behavior", "Use Camera DPI for All Imported Images", "Default TLV Caching Behavior", "Column Icon", "Palette Type on Loading Raster Image as Color Model", "New Levels Default to Current Camera Size", "Vector Snapping", "Use Arrow Key to Shift Cell Selection", "Enable to Input Cells without Double Clicking", "Show Toolbar in the XSheet", "Expand Function Editor Header...", "Show Column Numbers in Column Headers".

Thank you!

### PR DESCRIPTION
**Pull request by jcome**
- more UI strings avaiable for localization
- fix #1458 partly

### CODE REVIEW

File: toonz/sources/toonzlib/cleanupcolorstyles.cpp [hunk 1]
```diff
@@ -90,9 +90,9 @@ void TCleanupStyle::setColorParamValue(int index, const TPixel32 &color) {
 QString TCleanupStyle::getParamNames(int index) const {
   switch (index) {
   case 0:
-    return "Brightness";
+    return QObject::tr("Brightness");
   case 1:
-    return "Contrast";
+    return QObject::tr("Contrast");
```
Comment (shun-iwasawa) [inline toonz/sources/toonzlib/cleanupcolorstyles.cpp]: Thank you for the modification, it's very helpful.
Can you please make "HRange" and "Line Width" translatable as well? They are in `TColorCleanupStyle::getParamNames()` in [lines 227-231](https://github.com/opentoonz/opentoonz/blob/792dc2b6b402d622292efe815bfaa2635c43eeb9/toonz/sources/toonzlib/cleanupcolorstyles.cpp#L227-L231) of `cleanupcolorstyles.cpp`.

Comment (jcome) [reply toonz/sources/toonzlib/cleanupcolorstyles.cpp]: Thanks for your time to review and the kind words.
"HRange" and "Line Width" are added now.

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by shun-iwasawa** [comment]
LGTM Thank you @jcome !

**Pull request merged**